app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
+    load_and_authorize_resource through: :physical_ticket, singleton: true, shallow: true
```

Comment:
we need a ticket scanning object with physical ticket set before the create action is actually called so that we can define the ability for create action. With just authorize resource it won't be possible.